cf-java-monitoring-custom-metrics-clients/cf-custom-metrics-clients-java/src/main/java/com/sap/cloud/cf/monitoring/java/converter/MetricConverter.java

```diff
@@ -0,0 +1,43 @@
+package com.sap.cloud.cf.monitoring.java.converter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.sap.cloud.cf.monitoring.client.model.Metric;
+
+public abstract class MetricConverter<T> {
+    private static final String KEY_TYPE = "type";
+
+    enum MetricType {

```

Comment:
The default access modifier is enough here. It is the most restrictive option.

Reply:
The default access modifier is highly unusual in Java. It is even marked by tools like Sonatype or check style as a code smell. `protected` can be more restrictive, since it only allows access for instances of the same type (or subtype).

Reply:
done